Zend/tests/partial_function/variadics-named-nested-overwrite.phpt

```diff
@@ -0,0 +1,28 @@
+--TEST--
+Named variadics nested overwrite param.
+--FILE--
+<?php
+function variadic($one, ...$two) {
+    return [
+        func_num_args(), 
+        func_get_args(), 
+        [$one, $two]];
+}
+
+$variadic = variadic(?, three: 3);
+
+$variadic = 
+    $variadic(?, four: 4);
+
+$partial = $variadic(one: 1, two: [2], four: 5);
```

Comment:
I'm not sure this behavior is right. If we interpret partials "as if" you passed all the arguments directly, you'd have `$variadic(one: 1, four: 4, two: [2], four: 5)`, which would normally throw an `Error` due to duplicate parameter.

Reply:
It seemed useful in the case of partials specifically to allow overwriting a previously named parameter.

Reply:
FWIW, I think that behaviour is inconsistent with positional parameters, because you can't override positional ones after they've already been filled. I think I agree that it should throw an error in that case. Also, allowing overwriting seems leaky to me, for example if you wished to hide the fact that a certain parameter existed by prefilling it, then you'd have to contend with a caller possibly overriding that. (If I'm reading this correctly)